packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -637,7 +661,22 @@ contract("MerkleDistributor.js", function(accounts) {
         count++;
       }
       const average = total.divn(count);
-      assert.equal(Math.floor(average.toNumber()), 70418);
+      assert.equal(Math.floor(average.toNumber()), 70396);
+    });
+    it("(batch claim): gas average first 25", async function() {
+      for (let i = 0; i < 25; i++) {
+        const leaf = rewardLeafs[i];
+        const proof = merkleTree.getProof(leaf.leaf);
+        batchedClaims.push({
+          windowIndex: windowIndex,
+          account: leaf.account,
+          accountIndex: leaf.accountIndex,
+          amount: leaf.amount,
+          merkleProof: proof
+        });
+      }
+      const tx = await merkleDistributor.claimMulti(batchedClaims, [rando], [rewardToken.address]);
+      assert.equal(Math.floor(tx.receipt.gasUsed / batchedClaims.length), 30858);
```

Comment:
Compare to 38401 for #2672 [here](https://github.com/UMAprotocol/protocol/pull/2672/files#r590803497)